feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-base.component.ts

```diff
@@ -146,16 +146,32 @@ export class ConfiguratorAttributeBaseComponent {
    * @param expMode - Is expert mode set?
    * @param label - value label
    * @param techName - value technical name
+   * @param configuratorValue - item Object which contains the valuePrice
+   * @param isSelected - is item in dropdownlist selected
    */
   getLabel(
     expMode: boolean,
     label: string | undefined,
-    techName: string | undefined
+    techName: string | undefined,
+    configuratorValue?: Configurator.Value | undefined
   ): string {
     let title = label ? label : '';
     if (expMode && techName) {
       title += ` / [${techName}]`;
     }
+    if (configuratorValue && !configuratorValue.selected) {
+      if (
+        configuratorValue.valuePrice?.value !== undefined &&
+        configuratorValue.valuePrice.value < 0
+      ) {
+        title += ` [${configuratorValue.valuePrice?.formattedValue}]`;
+      } else if (
+        configuratorValue.valuePrice?.value !== undefined &&
```

Comment:
If you add `?` after `configuratorValue` and `valuePrice` then `configuratorValue?.valuePrice?.value` then `configuratorValue.valuePrice?.value !== undefined` is not neccessary

Reply:
done